src/components/TeaserFront/Tile.js

```diff
@@ -153,7 +157,18 @@ TeaserFrontTileRow.defaultProps = {
   columns: 1
 }
 
-const Tile = ({ children, attributes, image, alt, onClick, color, bgColor, align, aboveTheFold }) => {
+const Tile = ({
+  children,
+  attributes,
+  image,
+  alt,
+  onClick,
+  color,
+  bgColor,
+  align,
+  aboveTheFold,
+  imageOnly
```

Comment:
Tile already has an `showImage` option in the editor—`onlyImage` would therefore be more consistent